assets/src/edit-story/components/panels/layer/layer.js

```diff
@@ -52,9 +52,9 @@ const LayerButton = styled.button.attrs({
 
   ${({ isSelected, theme }) =>
     isSelected &&
-    `
-    background: ${rgba(theme.colors.accent.primary, 0.14)};
-  `}
+    css`
+      background: ${rgba(theme.colors.bg.v17, 1)};
```

Comment:
`rgba()` is only needed, when alpha is less than 1. For this, please just use the color directly:

```js
      background: ${theme.colors.fg.gray24};
```